content/blog/pulumi-loves-python/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Pulumi + Python: Bringing the Best of Modern Python to IaC"
```

Comment:
```suggestion
title: "Pulumi + Python: bringing the best of modern Python to IaC"
```

Feel free to ignore this and all other case changes if we decided against sentence casing!